examples/managementCli/package.json

```diff
@@ -29,6 +29,7 @@
     "@rollup/plugin-typescript": "^8.3.0",
     "dotenv": "^16.1.4",
     "rollup-plugin-delete": "^2.0.0",
-    "ts-node": "^10.8.2"
+    "ts-node": "^10.8.2",
+    "tsx": "^4.20.3"
```

Comment:
ts-node is having trouble running TypeScript files with ES modules, prefer to use tsx